src/current/v23.1/import-performance-best-practices.md

```diff
@@ -37,7 +37,7 @@ When importing into a new table, split your dump data into two files:
 1. A SQL file containing the table schema.
 1. A CSV, delimited, or AVRO file containing the table data.
 
-Convert the schema-only file using the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page). The Schema Conversion Tool automatically creates a new CockroachDB {{ site.data.products.serverless }} database with the converted schema. {% include cockroachcloud/migration/sct-self-hosted.md %}
+Convert the schema-only file using the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachcloud/migrations-page). The Schema Conversion Tool automatically creates a new CockroachDB {{ site.data.products.cloud }} database with the converted schema. {% include cockroachcloud/migration/sct-self-hosted.md %}
```

Comment:
Isn't it just a CockroachDB database? Cloud seems to be a deployment detail. For parity with the previous content, I think this should be Standard? That's how you have it in the next file down.

Reply:
No; this should be any Cloud database, because the SCT creates a new db from whichever Cloud cluster you're running it from. So if you're on Advanced, it's an Advanced database, etc.